### PR TITLE
Mobile website

### PR DESCRIPTION
**Pull request by Alexander-Schulte**

### CODE REVIEW

**Comment by Alexander-Schulte** [comment]
Hey Guys, here is my Website! Feel free to comment tips or problems in my code! Enjoy your evening

**Comment by hugh-burgess** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -1,43 +1,80 @@
...
+    <meta charset="utf-8" />
+    <title>Alexander Schulte</title>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to the website of Alexander Schulte</h1>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <li>Contact</li>
```
Comment (namirsab) [inline index.html]: I see these 2 navigation elements are not links, compared to the ones below. Is that intended?

File: index.html [hunk 2]
```diff
@@ -1,43 +1,80 @@
...
+            <a href="#Career">Career</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2>This is me!</h2>
+        <img
+          width="300"
```
Comment (namirsab) [inline index.html]: For now, this is fine as you don't have any other way to change the size of your images, but remember that this is not a good practice. 

You can read more about images here: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML#width_and_height

File: index.html [hunk 3]
```diff
@@ -1,43 +1,80 @@
...
+          height="300"
+          src="/images/AlexanderSchulte_Brighton.JPG"
+          alt="Alexander Schulte profile picture"
+        />
+
+        <h2 id="About Me">About Me</h2>
```
Comment (mpagels) [inline index.html]: This seems to be odd. Would recommend this:
```suggestion
        <h3 id="About Me">About Me</h3>
```
The headings present level of sections headings. h1 is the highest, h6 the lowest. You used the first h2 in line 25 for the "This is me!". I would assume that the "about me" line is something under the "This is me". I hope you understand what I mean ;-) See more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).

File: index.html [hunk 4]
```diff
@@ -1,43 +1,80 @@
...
+      <article>
+        <h2 id="Career">Career</h2>
+        <img
+          width="325"
+          height="400"
+          src="/images/Alexander_Schulte.jpg"
+          alt="Alexander Schulte football picture"
+        />
+
+        <h2>It´s football time!</h2>
```
Comment (mpagels) [inline index.html]: same

File: index.html [hunk 5]
```diff
@@ -1,43 +1,80 @@
...
+        <p>
+          Yeah as you know I like football very much.. I play football since I
+          was 3 years old. I startet in Dresden and played there for 11 years.
```
Comment (namirsab) [inline index.html]: You  have a typo here: `I started...`

**Review by namirsab — COMMENTED** [review]
Super good!  💪 
Just some little comments!

File: index.html [hunk 6]
```diff
@@ -1,43 +1,80 @@
...
+          With 14 I moved to Dortmund and played for Borussia Dortmund. Titles:
+        </p>
+
+        <ul>
+          <li>Bundsliga U19 Championship</li>
+        </ul>
+      </article>
+
+      <h2>That´s the End!</h2>
```
Comment (mpagels) [inline index.html]: the very same. ;-)

**Review by mpagels — CHANGES_REQUESTED** [review]
Very very good Alex. I found something regarding to headlines. Pls review.

Very good job.

**Comment by Alexander-Schulte** [comment]
Thank you for the feedback quys! Fixed all the things you commented. ✔️

**Review by mpagels — APPROVED** [review]

**Pull request closed without merging**